<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QWidget" name="Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>622</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QLCDNumber" name="waterNumber"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>USE DIRT</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="lava">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../textures.qrc">:/textures/lava.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="stone">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../textures.qrc">:/textures/stone.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="grassNumber"/>
    </item>
    <item row="1" column="5">
     <widget class="QLCDNumber" name="lavaNumber"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="dirt">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../textures.qrc">:/textures/dirt.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>USE GRASS</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="water">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../textures.qrc">:/textures/water.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="snow">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../textures.qrc">:/textures/snow.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="grass">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../textures.qrc">:/textures/grass.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="dirtNumber"/>
    </item>
    <item row="1" column="4">
     <widget class="QLCDNumber" name="snowNumber"/>
    </item>
    <item row="1" column="2">
     <widget class="QLCDNumber" name="stoneNumber"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>USE STONE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>USE WATER</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>USE SNOW</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>USE LAVA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>select block</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>select block</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string>select block</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_11">
      <property name="text">
       <string>select block</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="craftButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CRAFT IT --&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../textures.qrc"/>
 </resources>
 <connections/>
</ui>
